data/devices/huawei.yml

```diff
@@ -223,3 +223,31 @@
     recovery:
       - /dev/block/platform/hi_mci.0/by-name/recovery
       - /dev/block/mmcblk0p28
+      
+ - name: Huawei Y635
```

Comment:
@shahr773 Sorry, I meant here. The build failed because there's an extra space character right before `- name`.

![image](https://cloud.githubusercontent.com/assets/646253/25307167/407be5c0-276a-11e7-86af-624d38ce2981.png)